packages/cli-tools/bin/common.ts

```diff
@@ -90,4 +91,20 @@ export function createFnParseInt(name: string): (s: string) => number {
         }
         return n
     }
+}
+
+export const createStreamId = (streamIdOrPath: string|undefined, options: any) => {
```

Comment:
Maybe it's just me, but `createStreamId` conjures up an image of an object being created or something getting persisted / external API call. Something like `formStreamId` is more string-like, but dunno?

Reply:
Renamed to `getStreamId`. The original `createStreamId` name was used because the method creates a new piece of information instead of getting it from some object. But agree that it may be misleading in the sense you described.